site/docs/master/httpproxy.md

```diff
@@ -1416,3 +1417,4 @@ Some examples of invalid configurations that Contour provides statuses for:
  [9]: {% link docs/master/annotations.md %}
  [10]: /docs/{{site.latest}}/api/#projectcontour.io/v1.Service
  [11]: configuration.md#fallback-certificate
+ [12]: https://www.envoyproxy.io/docs/envoy/v1.11.2/api-v2/api/v2/route/route.proto.html#envoy-api-field-route-routeaction-idle-timeout
```

Comment:
It probably makes sense to use a versioned link here, but we should link to the latest version `1.14.2`.

Reply:
updated all the links on the page; I also realized there was already a link for the idle timeout (`[6]`) so de-duped.